web/views/quote_view.ex

```diff
@@ -1,3 +1,21 @@
 defmodule HonGraffitiPhoenix.QuoteView do
   use HonGraffitiPhoenix.Web, :view
+
+  def render("index.json", %{quotes: quotes}) do
+    %{
+      quotes: Enum.map(quotes, &quote_json/1)
```

Comment:
Yeah somethin' like that. Honestly was just imitating the style of the API guide I was following. I personally don't really care about this vs. `%{quotes: Enum.map(quotes, &quote_json/1)}`, if you have strong feelings either way I'm happy to follow those.